manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+    "manifest_version": 2,
+    "author": "Smilo Platform B.V.",
+    "name": "Smilo Platform: Wallet",
+    "description": "Smilo Platform Wallet allowing you to create, import, export wallets, see your funds and send funds.",
+    "version": "1.0",
+    "background": {
+        "scripts": [
+            "./src/webplugin/popup.ts"
```

Comment:
Shouldn't this be a Javascript file? You are referring to a Typescript file which still needs to be transpiled before use.